extide/gradle/apichanges.xml

```diff
@@ -83,6 +83,22 @@ is the proper place.
     <!-- ACTUAL CHANGES BEGIN HERE: -->
 
     <changes>
+        <change id="gradle-init-javaversion">
+            <api name="general"/>
+            <summary>Gradle InitOperation now Supports  --java-version and --comments flags</summary>
```

Comment:
is the `--java-verison` flag brand new? I can't find anything about that one. Is it only there to setup the project template or is more to that?

Reply:
Yes. It's new in maybe 8.5.
I have a new Wizard page on my design board. 
![image](https://github.com/apache/netbeans/assets/1381701/9c42223f-c49d-42a7-8d13-a146d03fd2e1)

Reply:
So its "only" a field passed to the template generator or is it more than that? (e.g does it download a JDK for gradle?)

Reply:
That just puts the following into `build.gradle`
```groovy
// Apply a specific Java toolchain to ease working on different environments.
java {
    toolchain {
        languageVersion = JavaLanguageVersion.of(22)
    }
}
```